.github/workflows/deploy_production.yml

```diff
@@ -0,0 +1,46 @@
+# Deploys to nypl-dams-prod.
+name: deploy production
+
+on:
+  push:
```

Comment:
Just a note: once we know this is working, this should probably change to `pull_request`. It'll force you and Alessandra to make pull requests for deployments but you should both decide if that's a good workflow.